### ISSUE TITLE
Should we consider merging this into upstream's ID2T?

### ISSUE THREAD

**Issue by UndeadKernel** [Trace-Share/ID2T#5]
Hey,
I'm the main designer of ID2T and I just noticed your fork from what I assume was a wrong pull request to our repository :D 
The new functionality you developed looks interesting and I was wondering if it would make sense to incorporate it into upstream ID2T.
Let me know what you think of this to coordinate the merging effort.

**Comment by cermmik** [comment]
Hi, thank you for your interest! The code is part of the bachelor thesis and is still being developed. We plan to complete it by the end of May. Currently, we are preparing additional tests and evaluation of the whole concept. Once we've verified it, we can discuss the incorporation of the code to the upstream. (I planned to write you an e-mail after that. )

From my point of view, some kind of research collaboration would be more interesting :) For example, we can try to perform some evaluation of both attack generation and trace manipulation approaches, and provide their pros and cons...

Our effort is part of the bigger "plan" focused on an evaluation of research on anomaly and attack detection. We published our ideas and plans in the paper [Towards Provable Network Traffic Measurement and Analysis via Semi-Labeled Trace Datasets](https://ieeexplore.ieee.org/document/8506498). So, if you are interested, we can discuss it in more detail via email or Skype.
